app/kubernetes/views/stacks/logs/logsController.js

```diff
@@ -58,7 +58,7 @@ class KubernetesStackLogsController {
       Pods: [],
     };
 
-    const promises = _.map(app.Pods, this.generateLogsPromise);
+    const promises = _.flatMap(_.map(app.Pods, (pod) => _.map(pod.Containers, (container) => this.generateLogsPromise(pod, container))));
```

Comment:
If i'm right you can use `_.flatMap(app.Pods, (pod) => _.map(pod.Containers, (container) => this.generateLogsPromise(pod, container)));`